### PR TITLE
Added filtering on datetime for queue element creation times

### PR DESCRIPTION
**Pull request by bakspace-itk**
Also reordered the elements and added some styling to keep everything looking good.

### CODE REVIEW

File: OpenOrchestrator/orchestrator/tabs/queue_tab.py [hunk 1]
```diff
@@ -70,17 +71,19 @@ def __init__(self, queue_name) -> None:
 
         with ui.dialog(value=True).props('full-width full-height') as dialog, ui.card():
             with ui.row().classes("w-full"):
-                self._create_column_filter()
+                self.from_input = DatetimeInput("From Date", on_change=self._update, allow_empty=True).style('margin-left: 20px')
```
Comment (ghbm-itk) [inline OpenOrchestrator/orchestrator/tabs/queue_tab.py:74]: Why do you define a margin?

File: OpenOrchestrator/orchestrator/tabs/queue_tab.py [hunk 2]
```diff
@@ -70,17 +71,19 @@ def __init__(self, queue_name) -> None:
...
                     options=[100, 200, 500, 1000, "All"],
                     label="Limit",
                     value=100,
                     on_change=self._update).classes("w-24")
 
-                ui.switch("Dense", on_change=lambda e: self._dense_table(e.value))
-
                 ui.space()
+
+                ui.switch("Dense", on_change=lambda e: self._dense_table(e.value))
```
Comment (ghbm-itk) [inline OpenOrchestrator/orchestrator/tabs/queue_tab.py:85]: I like that you moved this :)

File: changelog.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - 'Pausing' status added to triggers.
 - Disabling a 'Running' trigger in Orchestrator will set its status to 'Pausing'.
 - Scheduler will change a 'Pausing' trigger to 'Paused' when the process is done.
+- Queues can now be filtered by 'Created Date'
```
Comment (ghbm-itk) [inline changelog.md:16]:
```suggestion
- Queue elements can now be filtered by 'Created Date'.
```

File: tests/test_db_util.py [hunk 4]
```diff
@@ -160,6 +162,13 @@ def test_queue_elements(self):
         elements = db_util.get_queue_elements("Queue", reference="Foo")
         self.assertEqual(len(elements), 0)
 
+        # Filter by date
+        logs = db_util.get_queue_elements("Queue", from_date=creation_time)
+        self.assertEqual(len(logs), 2)
+
+        logs = db_util.get_queue_elements("Queue", to_date=creation_time)
+        self.assertEqual(len(logs), 0)
+
```
Comment (ghbm-itk) [inline tests/test_db_util.py:171]: Maybe a test that filters on both?

Comment (bakspace-itk) [reply tests/test_db_util.py:171]: Done!

File: OpenOrchestrator/orchestrator/tabs/queue_tab.py (hunk 1 above)
Comment (ghbm-itk) [reply OpenOrchestrator/orchestrator/tabs/queue_tab.py:74]: I can see in the css that the padding of a scroll area is 1rem (16px). If you set it to that it should also scale correctly.

```suggestion
                self.from_input = DatetimeInput("From Date", on_change=self._update, allow_empty=True).style('margin-left: 1rem')
```

**Review by ghbm-itk — APPROVED** [review]

**Pull request merged**